appveyor_mingw_cygwin.yml

```diff
@@ -52,13 +57,12 @@ build_script:
     - cd %APPVEYOR_BUILD_FOLDER%
     - cmd: if "%BUILD%"=="MINGW64" C:\msys64\usr\bin\bash -c "%CMD%"
     - cmd: set "PATH=c:\cygwin64\usr\local\bin;c:\cygwin64\bin;c:\cygwin64\usr\bin;c:\cygwin64\usr\sbin;"
-    - cmd: set  CMD=rm -rf build
+    - cmd: set  CMD=cmake.exe --version
```

Comment:
Remove since root cause was found?

Reply:
The `rm -rf build` code is pointless.   Reporting `cmake --version` is useful.

I've tried to submit a new PR and Git is going to fight me to the end.  Let's just get this working correctly and closed.